e2e/fixtures/rsbuild-csr-less/src/app/app.component.less

```diff
@@ -0,0 +1,8 @@
+@bg: black;
+@text: white;
+
+.my-p {
+  background-color: @bg;
+  color: @text;
+  padding: 10px;
+}
```

Comment:
Could be add more representative mock styles and include comments that elaborate why the code is here and what should work.

Reply:
I think this is representative enough - it uses less specific features for storing variables and using them.
comments can be added to explain that this is testing the functionality.